features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Overview.jsx

```diff
@@ -136,26 +141,66 @@ class Overview extends Component {
     handleTagChange(tags) {
         const api = new Api();
         const { apiUUID } = this;
+        const currentAPI = this.state.api;
         const promisedApi = api.get(apiUUID);
         promisedApi
-            .then((response) => {
-                const apiData = JSON.parse(response.data);
+            .then((getResponse) => {
+                const apiData = JSON.parse(getResponse.data);
```

Comment:
I think there is JSON parsed object in swagger client response object, so we don't need to call JSON.parse again, in this case, getResponse.obj will contain the JSON parsed response data object.

Reply:
Fixed.